pkg/memorystore/memorystore.go

```diff
@@ -158,7 +158,8 @@ func addParentsAndChildren(parentPath, path []string, secretConfig *v1alpha1.Sec
 // addAliasParentsAndChildren is a rangeFunc that sets the parents and children for alias dependency nodes
 //   all aliases should be parents of the relevant secret key
 func addAliasParentsAndChildren(parentPath, path []string, secretConfig *v1alpha1.SecretConfig, keyConfig *v1alpha1.KeyConfig, aliasConfig *v1alpha1.AliasConfig, nodes []*v1alpha1.Node) []*v1alpha1.Node {
-	if keyConfig.Type == v1alpha1.TypePKCS12 && aliasConfig != nil {
+	if (keyConfig.Type == v1alpha1.TypePKCS12 || keyConfig.Type == v1alpha1.TypeTrustStore) && aliasConfig != nil {
+		// if (keyConfig.Type == v1alpha1.TypePKCS12) && aliasConfig != nil {
```

Comment:
Spare comment here. Also, usually I would add a test case to the test rig to make sure this modification acts as I expected, but that can also be tedious.